Makefile

```diff
@@ -2,7 +2,8 @@
 # (c) 1998--2024 Martin Mares <mj@ucw.cz>
 
 OPT=-O2
-CFLAGS=$(OPT) -Wall -W -Wno-parentheses -Wstrict-prototypes -Wmissing-prototypes
+CFLAGS=$(OPT) -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -arch x86_64 -arch arm64 -Wall -W -Wno-parentheses -Wstrict-prototypes -Wmissing-prototypes
+TARGET_ARCH=-isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -arch x86_64 -arch arm64
```

Comment:
Why are `CFLAGS` and `TARGET_ARCH` the same?

I think these Makefiles need to still work for i386 and ppc and ppc64 architectures.
Maybe we need to check SDK versions to see which architectures can be built?
One method Makefile files or Configure files use to do that is to run a test compile/link command and check for error.

Reply:
done